workload/run.yml

```diff
@@ -0,0 +1,20 @@
+---
+# this is a sample of workload
+# prerequisite: 
+# - we rely on the rally template mechanism
+# 
+rally: 
+    enabled: true
```

Comment:
The purpose is to give the default value for all scenarios defined under the `scenarios` key.
In this example and according to the previous review we could omit it.